src/app-e-heat-input-from-oil-workspace/app-e-heat-input-from-oil.service.ts

```diff
@@ -98,12 +102,13 @@ export class AppEHeatInputFromOilWorkspaceService {
       updateDate: timestamp,
     });
 
-    await this.repository.save(entity);
+    await repository.save(entity);
     entity = await this.repository.getAppEHeatInputFromOilById(entity.id);
```

Comment:
This should be updated to use the transactional repository:
```suggestion
    entity = await repository.getAppEHeatInputFromOilById(entity.id);
```

Reply:
updated.